pkg/agent/secondarynetwork/init.go

```diff
@@ -48,6 +49,7 @@ func NewController(
 	podInformer cache.SharedIndexInformer,
 	podUpdateSubscriber channel.Subscriber,
 	secNetConfig *agentconfig.SecondaryNetworkConfig, ovsdb *ovsdb.OVSDB,
+	pIfacestore interfacestore.InterfaceStore,
```

Comment:
Nit: pIfacestore -> pIfaceStore
Probably better to put the argument after `podUpdateSubscriber` and before `secNetConfig`